### PR TITLE
Hide burndown graph placeholder if the user is not an admin of the project

### PR DESCRIPTION
**Pull request by bameda**
image

### CODE REVIEW

File: app/partials/backlog/backlog.jade [hunk 1]
```diff
@@ -11,7 +11,7 @@ div.wrapper(tg-backlog, ng-controller="BacklogController as ctrl",
         div.backlog-summary(tg-toggle-burndown-visibility)
             include ../includes/components/summary
 
-            div.empty-burndown(ng-if="showGraphPlaceholder")
+            div.empty-burndown(ng-if="showGraphPlaceholder && project.i_am_owner")
```
Comment (superalex) [inline app/partials/backlog/backlog.jade]: Shouldn't it be i_am_admin after the refactor?

Comment (bameda) [inline app/partials/backlog/backlog.jade]: No, right now we have this attr in back. I have to change it too in back and front...

**Comment by superalex** [comment]
Great! :+1:

**Comment by superalex** [comment]
Backported

**Pull request merged**